great_expectations/util.py

```diff
@@ -1673,7 +1674,7 @@ def get_context(
     ge_cloud_base_url: Optional[str] = None,
     ge_cloud_access_token: Optional[str] = None,
     ge_cloud_organization_id: Optional[str] = None,
-    ge_cloud_mode: bool = False,
+    ge_cloud_mode: bool = None,
```

Comment:
Sorry this should also be typed with Optional
```suggestion
    ge_cloud_mode: Optional[bool] = None,
```

Reply:
addressed in 1ab4ba8d1

Reply:
good catch. updated in 11823350a